Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	go test ./...
```

Comment:
One thing to take into account is to use [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)

```
.PHONY: test

test:
	go test ./... 
```

Reply:
Thanks for pointing this.
Fixed in 72bb8c064231023c31607fd5933c5a7c9c7aa425